user_coffee_shop_query.rb

```diff
@@ -0,0 +1,26 @@
+require_relative 'distance_calculator'
+
+class UserCoffeeShopQuery
+  attr_reader :distances
+
+  def initialize(coffee_shops, user)
+    @coffee_shops = coffee_shops
+    @user = user
+  end
+
+  def show_closest
```

Comment:
Why did you choose to add a public `show_closest` method on a query object ?

Reply:
You're right. It doesn't really make sense. Do you think maybe renaming to `result` is a better option?